src/pages/workspace/distanceRates/PolicyNewDistanceRatePage.tsx

```diff
@@ -0,0 +1,111 @@
+import type {StackScreenProps} from '@react-navigation/stack';
```

Comment:
Let's rename this file to `CreateDistanceRatePage` to be consistent with the other features, e.g. Categories and Tags